database/availabilities.py

```diff
@@ -0,0 +1,57 @@
+from bson import ObjectId
+
+from database import collection_availabilities
+from models.models import Availability
+
+"""
+db integrity:
+- schedule delete - delete all schedule's availabilities
+- operator delete - delete all operator's availabilities
+- add availability - check for no duplications - NOT CHECKING
+"""
+
+
+# Schedule and operator ids are optional
+def get_availabilities(schedule_id: str, operator_id: str):
```

Comment:
you can use more precise typing as both can be None
def get_availabilities(schedule_id: str | None, operator_id: str | None)